Exercise-HTML-CSS-1/index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title> Girl Develop it</title>
+
+        <!--Stylesheets-->
+        <link rel="stylesheet" href="css/bootstrap.css">
+        <link rel="stylesheet" href="css/bootstrap-grid.min.css">
+        <link rel="stylesheet" href="css/style.css">
```

Comment:
As you can see, you are only using two files here, but you have many more in `css/` directory. Remove the unused files